services/web/server/tests/unit/isolated/test_application_settings.py

```diff
@@ -2,44 +2,214 @@
 # pylint:disable=redefined-outer-name
 # pylint:disable=no-name-in-module
 
+import argparse
 import json
+import os
+import re
+from typing import Dict
 
 import pytest
+from aiohttp import web
 from simcore_service_webserver.application_settings import (
     APP_SETTINGS_KEY,
     ApplicationSettings,
+    convert_to_app_config,
     setup_settings,
 )
+from simcore_service_webserver.cli import parse, setup_parser
+
+# FIXTURES -----------------------------
 
 
 @pytest.fixture
-def settings(monkeypatch) -> ApplicationSettings:
-    monkeypatch.setenv("SC_VCS_URL", "git@github.com:ITISFoundation/osparc-simcore.git")
-    monkeypatch.setenv("SWARM_STACK_NAME", "simcore_stack")
-    monkeypatch.setenv("STACK_NAME", "invalid_env")
-    monkeypatch.setenv("WEBSERVER_MANUAL_MAIN_URL", "http://some_doc.org")
-    app = {}
+def mock_env_makefile(monkeypatch):
+    """envvars produced @Makefile (export)"""
+
+    # TODO: make dump-envs shall produce a file that we load here
```

Comment:
is this still valid?

Reply:
yes, it is a possible improvement: adding a Makefile recipe (``make dump-envs``) that creates a envfile with all env vars defined in the main ``Makefile``